apm-agent-plugins/apm-kafka-plugin/apm-kafka-base-plugin/src/main/java/co/elastic/apm/agent/kafka/KafkaProducerInstrumentation.java

```diff
@@ -66,36 +61,29 @@ public String getAdviceClassName() {
         return "co.elastic.apm.agent.kafka.KafkaProducerInstrumentation$KafkaProducerAdvice";
```

Comment:
[minor] use `getClass().getName()` to make it consistent with others, doing that with all `getAdviceClassName` implementations could be a good idea.